packages/features/auth/lib/next-auth-options.ts

```diff
@@ -373,10 +373,25 @@ export const AUTH_OPTIONS: AuthOptions = {
         const belongsToActiveTeam = checkIfUserBelongsToActiveTeam(existingUser);
         const { teams, ...existingUserWithoutTeamsField } = existingUser;
 
+        let isOrgAdmin;
+        // Check if the existingUser is an org admin
+        if (existingUser.organizationId) {
+          const orgAdminQuery = await prisma.membership.findFirst({
+            where: {
+              teamId: existingUser.organizationId,
+              userId: existingUser.id,
+              OR: [{ role: MembershipRole.ADMIN }, { role: MembershipRole.OWNER }],
+            },
+          });
+
+          isOrgAdmin = !!orgAdminQuery;
+        }
```

Comment:
CC: @zomars is another one of them cases where this should live on me query instead of session? Due to revalidation and caching? I swear we had this talk before with `belongsToActiveTeam`

Reply:
More specifically add it from here: 

https://github.com/calcom/cal.com/blob/main/packages/trpc/server/middlewares/sessionMiddleware.ts#L74-L79